src/agent/agent.clj

```diff
@@ -318,11 +318,11 @@ fi")
       [nil "webhook failed"])))
 
 (defn succeed-task-with-message [task message]
-  (webhook {:id (:id task) :status "success" :logs message
+  (webhook {:id (:id task) :status "success" :logs message :uploaded (:uploaded task)
             :tracing-context (:tracing-context task)}))
 
 (defn fail-task-with-message [task message]
-  (webhook {:id (:id task) :status "failure" :logs message
+  (webhook {:id (:id task) :status "failure" :logs message :uploaded (:uploaded task)
```

Comment:
now that I noticed, the `redacted` field shouldnt be there as well?

Reply:
Well, I think it doesn't make sense to redact errors from shell, do you disagree?